pyiron_atomistics/calphy/job.py

```diff
@@ -779,7 +772,7 @@ def collect_general_output(self):
             self._output["fe/backward/lambda"] = list(blambda)
 
             # get final structure
-            pyiron_atoms = self.project.create.structure.read(
+            pyiron_atoms = read(
```

Comment:
Ah, I had missed the project could be a base project instead of an atomistic one, so that's on me.  But if you using ase.read now, you will also need to do the conversion in line 781 again, I think.

Reply:
I have a fix for this in https://github.com/pyiron/pyiron_base/pull/1352